people_tool/templates/people_tool/partials/courses.html

```diff
@@ -6,14 +6,46 @@ <h3 class="breadcrumbs">
     {% verbatim %}
     <small><i class="fa fa-chevron-right"></i></small>
     <a href="#/">Search People</a>
-    <small><i class="fa fa-chevron-right"></i></small>
-    {{ personId }}
+
+    <span ng-if="selectedPersonInfo.name_first">
+        <small><i class="fa fa-chevron-right"></i></small>
+        {{selectedPersonInfo.name_last}},&nbsp;{{selectedPersonInfo.name_first}}'s Courses
+    </span>
     <span ng-if="operationInProgress"><i class="fa fa-refresh fa-spin"></i></span>
   </h3>
 </nav>
 
 <main>
+ <div ng-if='selectedPersonInfo' class="row">
+
+  <div class="col-md-12">
+      <div id="personInfoDT_wrapper" class="dataTables_wrapper form-inline no-footer">
 
+        <table id="personInfoDT"
+               class="table table-striped table-hover ng-isolate-scope dataTable no-footer person-info-table"
+               role="grid" aria-describedby="personInfoDT_info"
+               cellspacing="0">
+          <thead>
+            <tr role="row">
+                <th tabindex="0" id="lname" aria-controls="DataTables_Table_0" rowspan="1" colspan="1" aria-sort="ascending">Last Name</th>
+                <th tabindex="0" id="fname" rowspan="1" colspan="1">First Name</th>
+                <th tabindex="0" rowspan="1" colspan="1">ID</th>
+                <th rowspan="1" colspan="1">Email</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr role="row" class="odd">
+              <td class="sorting" tabindex="0">{{selectedPersonInfo.name_last}}</td>
+              <td class="sorting" tabindex="0">{{selectedPersonInfo.name_first}}</td>
+              <td class="sorting" tabindex="0">{{selectedPersonInfo.univ_id}}</td>
+              <td class="sorting" tabindex="0">{{selectedPersonInfo.email_address}}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+  </div>
+</div>
+<p></p>
```

Comment:
Seems like we could use an ID-tagged or class-tagged div for the above block and introduce a bottom margin to get the spacing we want, vs using a blank semantic tag to get things to display the way we want.